### ISSUE TITLE
Cannot run synthesize.sh with multiple arguments

### ISSUE THREAD

**Issue by vijay-murali** [capergroup/bayou#46]
Running Bayou at 24a905c.

```
./synthesize.sh --num_programs 1 --num_samples 100 example_inputs/TestIO.java 
usage: synthesize.sh [OPTION]... [FILE]
 -help,--help              print this message
 -p,--num_programs <arg>   the maximum number of programs to return
 -s,--num_samples <arg>    the number of asts to sample from the model
```

**Comment by vijay-murali** [comment]
Seems to be fixed in e1e96ed.

**Comment by rmbarnett-rice** [comment]
Yea, caught that in the last commit. My inability to count. Off by one in checking for max allowable params.
